package/repos/codeberg.py

```diff
@@ -140,11 +140,14 @@ def fetch_commits(self, package):
                 if not created:
                     break
 
-                collaborators.append(commit.user)
+                if commit.user and commit.user != "":
```

Comment:
Only checking `commit.user` should be enough right?

```suggestion
                if commit.user:
```